tests/src/JIT/HardwareIntrinsics/X86/Avx/Add.Double.cs

```diff
@@ -28,32 +28,41 @@ private static void AddDouble()
                 // Validates basic functionality works, using Unsafe.Read
                 test.RunBasicScenario_UnsafeRead();
 
-                // Validates basic functionality works, using Load
-                test.RunBasicScenario_Load();
+                if (Avx.IsSupported)
+                {
+                    // Validates basic functionality works, using Load
+                    test.RunBasicScenario_Load();
 
-                // Validates basic functionality works, using LoadAligned
-                test.RunBasicScenario_LoadAligned();
+                    // Validates basic functionality works, using LoadAligned
+                    test.RunBasicScenario_LoadAligned();
+                }
 
                 // Validates calling via reflection works, using Unsafe.Read
                 test.RunReflectionScenario_UnsafeRead();
 
-                // Validates calling via reflection works, using Load
-                test.RunReflectionScenario_Load();
+                if (Avx.IsSupported)
```

Comment:
How do you feel about moving this check inside the `test.RunReflectionScenario_Load` and `test.RunReflectionScenario_LoadAligned` methods instead?  From reading the code here, it is not obvious why we are checking `Avx.IsSupported`.  But if the check moved inside the methods, it would be obvious, because directly inside that if check, we are calling `Avx.Load`.